build.sbt

```diff
@@ -30,6 +25,7 @@ lazy val protocVersion = sparkMajorVer match {
 
 ThisBuild / scalaVersion := scalaVer
 ThisBuild / organization := "io.graphframes"
+ThisBuild / version := "0.9.1-SNAPSHOT"
```

Comment:
I think this is not needed actually. It may even break it. May you remove it at all?

Reply:
```suggestion
```